### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by bbhtt**
I haven't tested this but hopefully this fixes the build issue in https://github.com/flathub/xyz.mx_moment.moment/pull/5. I also updated all the dependencies.

Also, hopefully closes https://github.com/flathub/xyz.mx_moment.moment/issues/1 and closes https://github.com/flathub/xyz.mx_moment.moment/issues/3

### CODE REVIEW

**Comment by bbhtt** [comment]
bot, build

File: python3-modules.json [hunk 1]
```diff
@@ -0,0 +1,1257 @@
...
+        {
+            "name": "python3-mistune",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"mistune\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/3a/c7/b0a4413a4d9b7a4fda0d710fd90dba62375f0d0c4544e848dc7656757c0c/mistune-2.0.4-py2.py3-none-any.whl",
```
Comment (Newbytee) [inline python3-modules.json:652]: This is a no-go unfortunately. Mistune 2.0 changes the API drastically from 0.8 and Moment hasn't been changed to work with the new one. Furthermore, when I tried updating Moment to use Mistune 2.0 I ran into what seemed to be a bug with Mistune itself, and I ended up suspending the port given that Mistune 3.0 should be coming out soon.

**Review by Newbytee — CHANGES_REQUESTED** [review]
There might be other dependencies that need downgrading too, I didn't go through all. Other than that, thanks for looking into this!

File: python3-modules.json (hunk 1 above)
Comment (bbhtt) [reply python3-modules.json:652]: Ok thanks, I'll downgrade it. Any other deps to worry about or do you want me to go regenrate all of them acording to https://gitlab.com/mx-moment/moment/-/blob/v0.7.3/requirements.txt ?

Comment (Newbytee) [reply python3-modules.json:652]: I think generating according to that file sounds like a sensible way to do it.

Comment (bbhtt) [reply python3-modules.json]: Ok, done. Since this works I have split the commits for readability.

**Comment by Newbytee** [comment]
bot, build

**Comment by bbhtt** [comment]
> Started [test build 104067](https://buildbot.flathub.org/#/builders/32/builds/104067)

Looks like going by requirements.txt is a no-go, it missed a bunch of deps and placed matrix-nio first when it should be last. I'll do it manually and pin everything to the versions included in the current manifest now

**Comment by bbhtt** [comment]
There is a bit of dependency resolution issue with the original manifest. It includes two different versions of aiofiles https://github.com/flathub/xyz.mx_moment.moment/blob/75facbe29ee0bc88b52b20f3d2b1ebd2b1d20cc7/xyz.mx_moment.moment.yaml#L138 (0.4.0) and https://github.com/flathub/xyz.mx_moment.moment/blob/75facbe29ee0bc88b52b20f3d2b1ebd2b1d20cc7/python3-matrix-nio.json#L115 (0.6.0). There can be only one version and matrix-nio 0.18.7 needs at least 0.6.0 so I'll have to bump that up.

**Comment by Newbytee** [comment]
That should be fine. We're using aiofiles 0.8.0 with Moment in Alpine and that works okay.

**Comment by bbhtt** [comment]
bot, build

(Only one I couldn't fix was idna - has two different versions of it. Shouldn't be two much of an issue. Rest of them are fixed.)

**Review by Newbytee — APPROVED** [review]
Looks good now!

**Comment by Newbytee** [comment]
Thanks a bunch for working on this!

**Pull request merged**